### PR TITLE
Gift card history timeline 

### PR DESCRIPTION
**Pull request by Cloud11PL**
I want to merge this change because I want to add gift card history timeline.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://feature-gift-cards-post-mvp.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3954 Add Gift Card update history](https://app.clickup.com/t/2549495/SALEOR-3954)

File: src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    }
+  };
+
+  switch (event.type) {
+    case GiftCardEventsEnum.ACTIVATED:
+      return intl.formatMessage(
+        {
+          defaultMessage: "Gift card was activated by {activatedBy}",
+          description: "gift card history message"
+        },
```
Comment (orzechdev) [inline src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx:50]: Messeges in `getEventMessage` could be also imported from `messages.ts` file

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx (hunk 1 above)
Comment (Cloud11PL) [reply src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx:50]: Some - yes, but not all of them because some return jsx.

**Review by dominik-zeglen — APPROVED** [review]

File: src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx (hunk 1 above)
Comment (orzechdev) [reply src/giftCards/GiftCardUpdate/GiftCardHistory/GiftCardHistory.tsx:50]: Ok, but jsx is used only for variables, messages objects (with `defaultMessage` and `description`) can be defined in separate file

**Review by orzechdev — COMMENTED** [review]
https://github.com/saleor/saleor-dashboard/pull/1597#discussion_r750210326

**Review by orzechdev — APPROVED** [review]

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**